etchosts/etchosts.go

```diff
@@ -69,32 +68,12 @@ func Drop(path string) {
 
 // Build function
 // path is path to host file string required
-// IP, hostname, and domainname set main record leave empty for no master record
 // extraContent is an array of extra host records.
-func Build(path, IP, hostname, domainname string, extraContent []Record) error {
```

Comment:
fair; I was expecting that could be the case, so used a separate commit; I'll remove the second commit